test/options.ts

```diff
@@ -0,0 +1,138 @@
+// Copyright 2019 Google LLC
```

Comment:
nit: We should be consistent throughout this PR of the header (there's 2019, 2021), unless these files are copied, in which case we don't need to touch the header.